Common/File/AndroidStorage.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <vector>
+#include <string>
+
+#include "Common/File/DirListing.h"
+
+// To emphasize that Android storage mode strings are different, let's just use
+// an enum.
+enum class Android_OpenContentUriMode {
+	READ = 0,  // "r"
+	READ_WRITE = 1,  // "rw"
+	READ_WRITE_TRUNCATE = 2,  // "rwt"
+};
+
+#if PPSSPP_PLATFORM(ANDROID) && !defined(__LIBRETRO__)
+
+#include <jni.h>
+
+extern std::string g_extFilesDir;
```

Comment:
I don't think this is used anymore.

-[Unknown]

Reply:
Unless in scoped storage mode, where it's the private part on "external" storage that we can write to. See NativeApp.cpp:

```
if (System_GetPropertyBool(SYSPROP_ANDROID_SCOPED_STORAGE)) {
		g_Config.defaultCurrentDirectory = Path(g_extFilesDir);
```